code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1342,3 +1342,17 @@
 	L.unignore_slowdown(type)
 	L.Dizzy(0)
 	L.Jitter(0)
+
+/datum/reagent/medicine/AcetylCysteine
+	name = "AcetylCysteine"
```

Comment:
I think it'd be better to stylize as `Acetyl-Cysteine` for consistence (and I also find it easier to read). And to use lowercase name in path (`acetylcysteine` or `acetyl_cysteine`) for consistence as well.